cmd/lotus-miner/actor.go

```diff
@@ -513,7 +513,7 @@ var actorControlList = &cli.Command{
 			}
 			kstr := k.String()
 			if !cctx.Bool("verbose") {
-				kstr = kstr[:9] + "..."
+				kstr = kstr[:6] + "..."
```

Comment:
I think it would be a little safer here to check the length `k` and then print different things based on that. If the string is 9 characters or fewer, print the whole string. If it's 10 or longer, print the first 6 characters with the `...`.

Reply:
Added a check for length of the multisig address, and only printing the first 6 characters with `...` if its larger then 9: https://github.com/filecoin-project/lotus/pull/9241/commits/3e74819519815cba11613146cfa92480a178c90a. Else it will print everything.